pkg/client/message.go

```diff
@@ -13,6 +13,9 @@ limitations under the License.
 
 package client
 
+// MessageData is the message payload data type.
+type MessageData interface{}
+
```

Comment:
Why is `MessageData` better than `interface{}`?

Reply:
a. it's nicer
b. It should be easier to change this as we need, we can just implement a different MessageData when we have some stories that requires some other implementation.